lib/timezone/lookup/google.rb

```diff
@@ -16,6 +19,10 @@ def initialize(config)
       def lookup(lat,lng)
         response = client.get(url(lat,lng))
 
+        if response.code == '403'
+          raise(Timezone::Error::Google, response.body)
```

Comment:
This makes me nervous. I don't like including an entire HTTP response body as an error message. I would be happy to look at a PR that includes a logging/debugging configuration option:

```
def lookup(lat, lng)
  if response.body == '403'
    config.logger.debug("Timezone::Error::Google : #{response.body}") if config.debug?
    raise(Timezone::Error::Google, '403 Forbidden')
  end
end
```